cortex.sh

```diff
@@ -417,6 +436,8 @@ if [ "$arg1" = "install" ]; then
     exit 1
   elif [ "$arg2" = "" ]; then
     prompt_for_telemetry && install_eks && install_cortex && info
+  elif [ "$arg2" = "cortex" ]; then
```

Comment:
Can we move install/uninstall cortex to `dev/`? It would be nice to keep the api to `cortex.sh` as small as possible, and the user shouldn't need to run that, just us. It's probably ok to not even use manager, we can just call the `kubectl` commands directly since in dev we have `kubectl`